### ISSUE TITLE
Import once doesn't work with watch

### ISSUE THREAD

**Issue by sbuzonas** [maoberlehner/node-sass-magic-importer#107]
Using the importer with node-sass in watch mode will result in a file with only the contents of the changed file excluding every other import.

**Comment by maoberlehner** [comment]
I can reproduce that usage with `--watch` is buggy but I'm not sure if I can reproduce your specific problem.

I created the following file:

```scss
@import '{ .a-class } from _file.scss';
@import '_file2.scss';

.thing {
    content: 'thing';
}
```

and compiled with:

`node-sass --importer node_modules/node-sass-magic-importer/dist/cli.js -o dist test.scss -w`

In watch mode, only the contents of `_file.scss` are imported. `_file2.scss` is ignored completely. Without watch mode everything works as expected.

I will look into this problem, maybe this will fix your problem too – otherwise please add a minimal test case how I can reproduce your specific problem.

Thank you for reporting this bug.

**Comment by sbuzonas** [comment]
That's my problem. I wasn't using the partial imports, so I only got the contents of the file that had changed.

I'm working around it with a custom importer for watch mode

```javascript
const importer = require('node-sass-magic-importer');

module.exports = importer({
  disableImportOnce: true
});
```

I couldn't figure out how to correctly solve it.

**Comment by maoberlehner** [comment]
I published a new release (4.1.5) which fixes this bug. Thank you very much for reporting the bug – because of this bug report, I could finally figure out whats wrong with running `node-sass-magic-importer` in combination with webpack and I also could fix this bug: https://github.com/maoberlehner/node-sass-magic-importer/issues/87
